serverless_k8s/README.md

```diff
@@ -0,0 +1,60 @@
+# Serverless Kubernetes Track
+Serverless Kubernetes is a track intended for benchmarking serverless Elasticsearch using a Kubernetes pod metrics dataset. A small corpus of Kibana object definitions is included for indexing with fast refresh enabled.
+
+## Track Parameters
+
+| Setting | Default | Description |
+| --- | --- | --- |
+| `bulk_indexing_clients` | `8` | The number of bulk indexing clients. |
```

Comment:
Is this what you normally run for tests vs. stateless?

Just curious because we are introducing quite a bit of "wait" time on the indexing pathway (waiting for object store flush) and I am wondering if that is a visible blocker with just 8 clients.

Reply:
Using a fixed number of clients simulates a closed system whereby the throughput is bound by the service time of each of the requests, but the value of 8 is just a default. 

We often increase or decrease it depending on the intention behind the benchmark, and if 8 isn't enough to 'induce' the behaviour/scenarios we want to benchmark then we can increase it at runtime (or the default here).